public/index.html

```diff
@@ -24,6 +24,7 @@
     <link href="https://fonts.googleapis.com/css?family=Exo+2|Open+Sans" rel="stylesheet" />
 
     <title>Authors Haven</title>
+    <base href="/" />
```

Comment:
What problem is the base tag solving??

Reply:
>The  __`<base>`__ element specifies the base URL to use for all relative URLs contained within a document and there can be only one  __`<base>`__ element in a document. 

Since we are building an SPA, we only have one Page/document, Instead of making it global  would prefixing all your pagination links with `/` be too much work??